flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/sinks/PrintTableSink.java

```diff
@@ -0,0 +1,63 @@
+package org.apache.flink.table.sinks;
+
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.typeutils.RowTypeInfo;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.datastream.DataStreamSink;
+import org.apache.flink.streaming.api.functions.sink.PrintSinkFunction;
+import org.apache.flink.table.utils.TableConnectorUtils;
+
+/**
+ * A simple {@link TableSink} to emit data to the standard output stream.
+ */
+public class PrintTableSink implements BatchTableSink , AppendStreamTableSink {
+
+	private String[] fieldNames;
+	private TypeInformation<?>[] fieldTypes;
+
+	@Override
+	public void emitDataSet(DataSet dataSet) {
+		try {
+			dataSet.print();
```

Comment:
This triggers an execution immediately. And does not only add a sink.

Reply:
Yeah , `print`  triggers an execution ,the method `emitDataSet `  is  for DataSet.

Reply:
Hi , do you have any better suggestions?
![printtable1](https://user-images.githubusercontent.com/18002496/60959543-f4ae3900-a33a-11e9-8048-746983768449.jpg)

Reply:
@twalthr Hi , do you have any better suggestions for `BatchTableSink`?